packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.base.tsx

```diff
@@ -131,11 +169,38 @@ export class ChoiceGroupOptionBase extends BaseComponent<IChoiceGroupOptionProps
     );
   };
 
+  private _onLabelWrapperRef(element: HTMLDivElement | null): void {
+    if (element) {
+      this._labelWrapperElementList.push(element);
+    }
+  }
+
+  private _calculateLabelTextOverflow = (): void => {
+    if (!this._labelWrapperElementList) {
+      return;
+    }
+    for (let labelListIndex = 0; labelListIndex < this._labelWrapperElementList.length; labelListIndex++) {
+      const labelElement = this._labelWrapperElementList[labelListIndex];
+      if (labelElement && hasOverflow(labelElement)) {
+        // This state allows adding an ellipsis to the labelWrapper if text is overflowing
+        this.setState({
+          hasOverflow: true
+        });
+      }
+    }
+  };
+
   private _onRenderLabel = (props: IChoiceGroupOptionProps): JSX.Element => {
+    const imageSize = props.imageSize;
+    const paddingBorderTop = 28; // includes padding, border, margin from styles
```

Comment:
The source code here should not be making any assumptions about styling, which is defined in another file.